code/modules/admin/fun_verbs.dm

```diff
@@ -910,7 +910,7 @@
 	message_admins("[ADMIN_TPMONTY(usr)] changed hivenumber of [ADMIN_TPMONTY(X)] from [hivenumber_status] to [newhive].")
 
 
-/datum/admins/proc/release()
+/datum/admins/proc/release(obj/OB in world)
```

Comment:
`OB` isn't used in the proc. How does this work?

Reply:
it doesn't matter what it's named, since it's being added as a verb, the argument specifies what it works on